documentation/CAMARA-Security-Interoperability.md

```diff
@@ -0,0 +1,283 @@
+Therefore scopes should be available to API implementations.
+
+---
+
+
+## Missing "openid" scope
+
+[OIDC Core Authentication Request](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest) states the following about the value of scope.
+> REQUIRED. OpenID Connect requests MUST contain the openid scope value. If the openid scope value is not present, the behavior is entirely unspecified. Other scope values MAY be present. ...
+
+This document defines that the authentication server SHOULD not return an id token if `openid` is missing in the scope parameter.
+
+This document defines the following error handling for a missing "openid" value in scope. 
+Please refer to [Authentication Error Response](https://openid.net/specs/openid-connect-core-1_0.html#AuthError).
+
+If "openid" is missing in the scope value but a claim that is [standardized in OIDC](https://openid.net/specs/openid-connect-core-1_0.html#Claims) is requested, then the Authorization Server returns an HTTP response code of 400 (Bad Request) and an error invalid_request.
+
+Clients SHOULD follow the OIDC standard and SHOULD include `openid` in the list of requested scopes.
+Without id token there is no `sub` field and the privacy features of OIDC are severely crippled. 
+Globally unique identifiers, like the MSISDN, should be avoided for privacy reasons.
+
+## Purpose
+
+In Camara `purpose` is used to convey to the user for which purpose an API is used. E.g.: Does the client request the user's location because it wants to show relevant tourist information or is the user's location information protecting the user when they withdraw money from an ATM? 
+
+Purpose has always to be reflected and audited when personal information is accessed or managed in certain legislations (e.g. in EU GDPR). To ensure interoperability across different legislations this profile requires that the purpose must always be declared within the supported OIDC or OAuth2 token requests if personal information is accessed or managed.
+
+### Purpose as a scope
+
+Purpose is one of the scope parameter's values. There MUST be exactly one purpose.
+
+The Authorization Server identifies the purpose using the prefix `dpv:`.
+
+This scope MUST have the following format: `dpv:<dpvValue>` where `<dpvValue>` is coming from [W3C DPV purpose definition](https://w3c.github.io/dpv/dpv/#vocab-purpose)
+
+
+
+### Outlook on purpose-handling leveraging Rich Authorization Request
+
+[RFC 9396 OAuth 2.0 Rich Authorization Requests](https://www.rfc-editor.org/rfc/rfc9396) defines a OAuth2 parameter `authorization_details` that is used to carry fine-grained authorization data. The parameter `authorization_details` can be used in all places where OAuth2 allows the `scope` parameter. That means that the parameter `authorization_details` can be used in all places where OIDC and CIBA allow the `scope` parameter.
```

Comment:
```suggestion
[RFC 9396 OAuth 2.0 Rich Authorization Requests](https://www.rfc-editor.org/rfc/rfc9396) defines a OAuth2 parameter `authorization_details` that is used to carry fine-grained authorization data. The parameter `authorization_details` can be used in all places where OAuth2 allows the `scope` parameter. That means that the parameter `authorization_details` can be used in all places where OIDC or OAuth2 allow the `scope` parameter.
```

Reply:
RAR says that authorization_details can be used in all places scope can be used. RAR does not mention CIBA.
But, yes, we can mention CIBA here.

Reply:
@AxelNennker the suggestion is to replace "OIDC and CIBA" by "OIDC or OAuth2", so NOT to mention CIBA. The suggestion makes sense for me, but propose to do it a later editorial PR. No need to dismiss the already done approvals for that.